architecture/infrastructure_components/web_console.adoc

```diff
@@ -15,17 +15,16 @@ Developers can use the web console to visualize, browse, and manage the contents
 of link:../../dev_guide/projects.html[projects].
 
 The web console is started as part of the
-link:kubernetes_infrastructure.html#master[master] and by default listens one
-port higher than the API server. All static assets required to run the web
+link:kubernetes_infrastructure.html#master[master]. All static assets required to run the web
 console are served from the `openshift` binary.
 
 When you access the web console from a browser, it first loads all required
-static assets. It then makes requests to the OpenShift and Kubernetes APIs using
-the values defined from the `openshift start` options `--public-kubernetes` and
```

Comment:
Openshift serves as proxy for kube requests; I don't think there's any use case for `--public-kubernetes` at this point (if there ever was).